go/kbfs/kbfsblock/id_test.go

```diff
@@ -75,7 +75,7 @@ func TestRandomIDInRange(t *testing.T) {
 		for j := i * 2; j < (math.MaxUint64 / 2); j *= 2 {
 			iAsFloat := float64(i) / maxUintFloat
 			jAsFloat := float64(j) / maxUintFloat
-			id, err := MakeRandomIDInRange(iAsFloat, jAsFloat)
+			id, err := MakeRandomIDInRange(iAsFloat, jAsFloat, false)
```

Comment:
Might as well make this `true` so it's deterministic as well (and seed `rand` at the beginning of the test).